### PR TITLE
build: Build movement-controller image in CI

### PR DESCRIPTION
**Pull request by froesef**
This PR will always build the movement-controller image that is used in some demos.

### CODE REVIEW

File: build/Makefile [hunk 1]
```diff
@@ -10,7 +10,15 @@ IMG_MC ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/movement-controller:${DOCKER_TA
 IMG_DM ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/dummy-mover:${DOCKER_TAGNAME}
 
 .PHONY: all
-all: docker-movement-controller docker-dummy-mover
+all: docker-movement-controller docker-build-dummy-mover
```
Comment (hunchback) [inline build/Makefile:13]: why not just append `docker-push-all` at the end of this line?

File: build/Makefile [hunk 2]
```diff
@@ -10,7 +10,15 @@ IMG_MC ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/movement-controller:${DOCKER_TA
...
+	$(MAKE) docker-push IMG=$(IMG_MC)
+	$(MAKE) docker-push IMG=$(IMG_DM)
```
Comment (hunchback) [inline build/Makefile:15]: As an alternative implement `all` as:
```
all: docker-build-all docker-push-all
```

File: build/Makefile [hunk 3]
```diff
@@ -10,7 +10,15 @@ IMG_MC ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/movement-controller:${DOCKER_TA
...
+
+.PHONY: docker-build-all
+docker-build-all: docker-movement-controller docker-build-dummy-mover
```
Comment (hunchback) [inline build/Makefile:18]: renaming to `docker-build` may be better

File: build/Makefile [hunk 4]
```diff
@@ -10,7 +10,15 @@ IMG_MC ?= ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/movement-controller:${DOCKER_TA
...
+
+.PHONY: docker-push-all
+docker-push-all:
```
Comment (hunchback) [inline build/Makefile:21]: renaming to `docker-push` may be better

**Review by hunchback — APPROVED** [review]
added some comments which are optional

File: build/Makefile (hunk 3 above)
Comment (froesef) [reply build/Makefile:18]: `docker-build` and `docker-push` are already inherited from docker.mk. If I specify the targets again they will be overwritten.

**Comment by roee88** [comment]
Can you explain why? I thought that you only use it for local testing and otherwise it is bundled in the manager

**Comment by froesef** [comment]
We're using the image quite a bit for internal demos. And like this it's easy to access it. Also it can be very useful for the mover development as it helps to deploy a minimal setup to quickly test changes to the motion controller in a K8s integrated scenario.

**Pull request merged**